app/sagas/deepLinking.ts

```
@@ -0,0 +1,397 @@
+				}
+			}
+			yield* put(loginRequest({ resume: params.token }, true));
+			yield* take(types.LOGIN.SUCCESS);
+			yield* put(appReady());
+			// Wait for the login saga's appStart(ROOT_INSIDE) before navigating, so
+			// InsideStack is mounted and goRoom dispatches into the correct stack.
+			const currentRoot = yield* appSelector(state => state.app.root);
+			if (currentRoot !== RootEnum.ROOT_INSIDE) {
+				yield* take(
+					(action: any) => action.type === types.APP.START && action.root === RootEnum.ROOT_INSIDE
+				);
```

ESLint

[error] Replace `⏎↹↹↹↹↹(action:·any)·=>·action.type·===·types.APP.START·&&·action.root·===·RootEnum.ROOT_INSIDE⏎↹↹↹↹` with `(action:·any)·=>·action.type·===·types.APP.START·&&·action.root·===·RootEnum.ROOT_INSIDE`

(prettier/prettier)